README.md

```diff
@@ -2,5 +2,106 @@
 
 [![Travis Build Status](https://travis-ci.org/logstash-plugins/logstash-output-rabbitmq.svg)](https://travis-ci.org/logstash-plugins/logstash-output-rabbitmq)
 
-This is a mixin supporting  the logstash rabbitmq inputs and outputs. If you plan on writing a logstash plugin using
-rabbitmq this may be useful.
+This is a plugin for [Logstash](https://github.com/elastic/logstash).
+
+It is fully free and fully open source. The license is Apache 2.0, meaning you are pretty much free to use it however you want in whatever way.
+
+
+## RabbitMQ Output Plugin Has Moved
+
+This RabbitMQ Output Plugin is now a part of the [RabbitMQ Integration Plugin][integration-source];
+this project remains open for backports of fixes from that project to the 5.x series where possible, but issues should first be filed on the [integration plugin][integration-issues].
+
+[integration-source]: https://github.com/logstash-plugins/logstash-integration-rabbitmq
+[integration-issues]: https://github.com/logstash-plugins/logstash-integration-rabbitmq/issues/
+
+## Documentation
+
+Logstash provides infrastructure to automatically generate documentation for this plugin. We use the asciidoc format to write documentation so any comments in the source code will be first converted into asciidoc and then into html. All plugin documentation are placed under one [central location](http://www.elastic.co/guide/en/logstash/current/).
+
+- For formatting code or config example, you can use the asciidoc `[source,ruby]` directive
+- For more asciidoc formatting tips, see the excellent reference here https://github.com/elastic/docs#asciidoc-guide
+
+## Need Help?
+
+Need help? Try #logstash on freenode IRC or the https://discuss.elastic.co/c/logstash discussion forum.
+
+## Developing
+
+### 1. Plugin Developement and Testing
+
+#### Code
+- To get started, you'll need JRuby with the Bundler gem installed.
+
+- Create a new plugin or clone and existing from the GitHub [logstash-plugins](https://github.com/logstash-plugins) organization. We also provide [example plugins](https://github.com/logstash-plugins?query=example).
+
+- Install dependencies
+```sh
+bundle install
+```
+
+#### Test
+
+- Update your dependencies
+
+```sh
+bundle install
+```
+
+- Run tests
+
+```sh
+bundle exec rspec
+```
+
+### 2. Running your unpublished Plugin in Logstash
+
+#### 2.1 Run in a local Logstash clone
+
+- Edit Logstash `Gemfile` and add the local plugin path, for example:
+```ruby
+gem "logstash-filter-awesome", :path => "/your/local/logstash-filter-awesome"
```

Comment:
I don't know if worthwhile to use this plugin's name, or keeping the `logstash-filter-awesome`